tools/CreateRuleFabricBot/CreateRuleFabricBot/CreateRuleFabricBot.csproj

```diff
@@ -1,8 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <NoWarn>CS8002;NU5105</NoWarn>
```

Comment:
Usually a good idea of adding `$(NoWarn)` to the list incase there are other shared warnings.